at-vuongtran/W1D4/b2.rb

```diff
@@ -0,0 +1,9 @@
+class Array
+  def sum
+    each do | x |
+      return nil if not (x.is_a?(Integer))
```

Comment:
ok nhưng sau này dùng ! thay cho not nhé, nó gọn hơn.
Ví dụ: `and` -> &&
           `or`    -> ||
           `not`  -> !